contrib/completion/bash/_todo

```diff
@@ -1,3 +1,21 @@
+_todo_get_ids()
+{
+	local pattern='^\s*"id": \([0-9]\+\),'
+	todo --porcelain list | grep -o "${pattern}" | sed -e "s/${pattern}/\1/" | sort | uniq
```

Comment:
Is `uniq` necesary here?

Reply:
No, you're right, that can be omitted here. `id` should be unique anyway.